packages/calcite-components/src/components/autocomplete-item/resources.ts

```diff
@@ -0,0 +1,17 @@
+import { Scale } from "../interfaces";
+
+export const CSS = {
+  container: "container",
+  containerActive: "container--active",
+  contentCenter: "content-center",
+  description: "description",
+  heading: "heading",
+  iconEnd: "icon-end",
+  iconStart: "icon-start",
+  scale: (scale: Scale) => `scale--${scale}` as const,
```

Comment:
Its used in `combobox-item`. 

https://github.com/Esri/calcite-design-system/blob/e0f5a85b34addaab150be07c1dc427a49689b6cf/packages/calcite-components/src/components/combobox-item/resources.ts#L13

I copied it from there. Is that ok?

Reply:
Gotcha, yeah this was just a sidebar; I  was curious if that's something we should be doing instead of using the attribute selector like this:

https://github.com/Esri/calcite-design-system/blob/dev/packages/calcite-components/src/components/icon/icon.scss#L18-L37

I think either are fine, I just hadn't seen the `scale--s` pattern in our codebase yet. After a quick grep I also found instances of `scale-s`, e.g.

https://github.com/Esri/calcite-design-system/blob/dev/packages/calcite-components/src/components/tab/tab.scss#L29-L42

Do we have any conventions on whether we should be using BEM in our CSS class names @jcfranco? It seems rather inconsistent between components.

Reply:
I agree `.scale--s` would be better than `.scale-s` to align with a standardized pattern. We don't really need BEM for specificity since our components are shadowed. Probably an issue to cleanup class names.

I do prefer classes over using attribute selectors because it forces those attributes to be reflected and can make some selectors quite complex.

Reply:
> I agree `.scale--s` would be better than `.scale-s` to align with a standardized pattern. We don't really need BEM for specificity since our components are shadowed. Probably an issue to cleanup class names.

 I also prefer `.scale--s` since it follows convention. I don't think it's worth an issue because the classes aren't exposed to users. But maybe we can recommend the `block--modifier-value` naming convention for future PRs so the codebase doesn't diverge any further, cc @jcfranco.

> I do prefer classes over using attribute selectors because it forces those attributes to be reflected and can make some selectors quite complex.

Yeah makes sense, thanks!